portal/views/partials/address/address-tokens.njk

```diff
@@ -69,3 +69,22 @@
     </div>
   </div>
 </div>
+
+
+<div class="block-section address-tokens isMobile">
```

Comment:
we already have a test for isMobile , if you have to use this css class use underscore syntax like the rest of the css. i.e: is_mobile